protocol/messenger_share_urls_test.go

```diff
@@ -0,0 +1,411 @@
+package protocol
```

Comment:
Is it possible to have some tests that parse expected url strings? This way we can ensure breaking changes to functionality gets addressed.

Reply:
This is actually quite close to what I was thinking.

`url := fmt.Sprintf("%s/c/%s#%s", baseShareURL, communityData, signature)`